<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QSpinBox" name="percentHeightSpinBox">
        <property name="toolTip">
         <string>height</string>
        </property>
        <property name="suffix">
         <string> %</string>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QSlider" name="widthSlider">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="sliderPosition">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="percentWidthSpinBox">
        <property name="toolTip">
         <string>width</string>
        </property>
        <property name="suffix">
         <string> %</string>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSlider" name="heightSlider">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="sliderPosition">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSpinBox" name="pixelWidthSpinBox">
        <property name="prefix">
         <string>width </string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="pixelHeightSpinBox">
        <property name="prefix">
         <string>height </string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="pixelComboBox">
     <item>
      <property name="text">
       <string notr="true">VGA (640x480)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">XGA (1024x768)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">NTSC (576x486)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">PAL (720x486)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">HD (1920x1080)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Flickr large (768x1024)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">iPhone landscape (480x320)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">160x160</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">HD 720 (1280x720)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">SVGA (800x600)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">PSP (480x272)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">2K (2048x1080)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="nothingRadioButton">
     <property name="text">
      <string>change nothing</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QComboBox" name="sizeRatioComboBox">
     <item>
      <property name="text">
       <string notr="true">1:1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">5:4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">4:3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">16:9</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QRadioButton" name="percentRadioButton">
     <property name="text">
      <string>resize percentaged</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QRadioButton" name="pixelRadioButton">
     <property name="text">
      <string>resize to pixels</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>widthSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>percentWidthSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>heightSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>percentHeightSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>percentWidthSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widthSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>percentHeightSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>heightSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
